ludwig/trainers/trainer_llm.py

```diff
@@ -28,6 +28,7 @@
 logger = logging.getLogger(__name__)
 
 
+###SXK do we want to have a spark trainer as well
```

Comment:
This would be a pretty big Ludwig contribution, since today we only have support for local and Ray backends for all parts of Ludwig. Would be a great contribution if you're interested in adding it!